packages/manager/modules/bm-server-components/src/server-dashboard/dashboard.html

```diff
@@ -139,6 +142,7 @@
                 data-ola="$ctrl.ola"
                 data-go-to-update-reverse-dns="$ctrl.goToUpdateReverseDns"
                 data-go-to-delete-reverse-dns="$ctrl.goToDeleteReverseDns"
+                data-state-prefix="$ctrl.statePrefix"
```

Comment:
still necessary ?

Reply:
Yes, because it isn't a child we go to interface page and it's interface children.